gspn.py

```diff
@@ -33,9 +32,13 @@ def __init__(self):
         self.__sparse_matrix_out = None
 
     def add_places(self, name, ntokens=None, set_initial_marking=True):
-        """
+        '''
+        Adds new places to the existing ones in the GSPN object. Replaces the ones with the same name.
 
-        """
+        :param name: (list str) denoting the name of the places
+        :param ntokens: (list int) denotes the maximum number of tokens of each place
```

Comment:
not the maximum number of tokens, but the current number of tokens of the given places. can be set as the initial marking depending whether the set_initial_marking parameter is True or False.